### PR TITLE
Feature/angular16

### PR DESCRIPTION
**Pull request by Boris-Bodin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] Other... Please describe: update dependencies
```

## What is the current behavior?

Peer dependency to "rxjs" (requires "~7.5.0", would install "7.8.0"), and Angular 15

## What is the new behavior?

Peer dependency Angular 16 only

## Does this PR introduce a breaking change?

```
[x] Yes
[ ] No
```

Need update to Angular 16

## Other information

Update nx to version 17

### CODE REVIEW

File: libs/ngx-localstorage/package.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   "name": "ngx-localstorage",
   "version": "5.0.0",
   "peerDependencies": {
-    "@angular/common": "^15.0.0",
-    "@angular/core": "^15.0.0"
+    "@angular/common": "^16.0.0",
+    "@angular/core": "^16.0.0"
```
Comment (bohoffi) [inline libs/ngx-localstorage/package.json]: As there aren't any breaking changes in the Angular API used in this library - what do you think about setting the version to `>=15.0.0`?

Comment (Boris-Bodin) [reply libs/ngx-localstorage/package.json]: Good idea, it will leave the lib usable in Angular 17 or later if there are no changes.

Comment (bohoffi) [reply libs/ngx-localstorage/package.json]: Exactly :)

**Review by bohoffi — APPROVED** [review]

**Comment by bohoffi** [comment]
You have to run `npm install` again to sync `package.json` and `package-lock.json` and commit again

**Comment by Boris-Bodin** [comment]
Thx, it's done

**Review by bohoffi — APPROVED** [review]

**Comment by bohoffi** [comment]
Thank you very much for your contribution - I'll prepare a new release

**Pull request merged**